gradle/buildScanUserData.gradle

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+def run(String commandLine) {
+    def process = commandLine.execute([], rootDir)
+    assert process.waitFor() == 0
```

Comment:
This is potentially a problematic code.

If I remember correctly, `waitFor()` will wait till the process ends, BUT also freeze when the process output is not consumed. You consume process output in the following line.

So if the process produces a lot of output, this will block the JVM execution forever.